screens/backupAndRestore/BackupAndRestoreScreen.tsx

```diff
@@ -160,7 +160,7 @@ const BackupAndRestoreScreen: React.FC<BackupAndRestoreProps> = props => {
             }>
             {restoreController.isBackUpRestoring
               ? t('restoreInProgress')
-              : t('restoreInfo')}
+              : t('restoreInfo', {driveName: getDriveName()})}
```

Comment:
drive seems more aligned to "Google Drive" , should we rename this to something like "cloudProvider" or "cloudSource"

Reply:
Will do it in next PR as this is sort of blocking the APK/IPA testing for this release.